coderadar-vcs/src/main/java/io/reflectoring/coderadar/vcs/RevCommitHelper.java

```diff
@@ -0,0 +1,38 @@
+package io.reflectoring.coderadar.vcs;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.revwalk.RevCommit;
+import org.eclipse.jgit.storage.file.FileRepositoryBuilder;
+
+public class RevCommitHelper {
```

Comment:
Why not also use the helper in the `ExtractProjectCommitsAdapter` ?

Reply:
Done.